src/components/common/Button.jsx

```diff
@@ -0,0 +1,9 @@
+import React from "react"
+
+export default function Button({type, children}) {
```

Comment:
encapsulate file scss

Reply:
styling for Button component has been encapsulated into a single stylesheet under its root folder
<img width="138" alt="image" src="https://github.com/thomasdinh2k/FintTestReact-LoginScreen/assets/130777277/175dc52b-8e89-4c34-9c21-b7a091c67d84">